lib/stalePullRequest.js

```diff
@@ -123,3 +130,38 @@ stalePullRequest.foundStopComment = function (commentsJsonResponse) {
 
     return false;
 };
+
+stalePullRequest.getCommits = function (commitsUrl, repositorySettings) {
```

Comment:
These are private, correct? Add the underscore, `_getCommits`. Otherwise, document them.